src/sagemaker/djl_inference/defaults.py

```diff
@@ -0,0 +1,45 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""Placeholder docstring"""
+from __future__ import absolute_import
+
+DEEPSPEED_RECOMMENDED_ARCHITECTURES = {
```

Comment:
stable diffusion not in the list?

Reply:
Good question, I think we need to clarify on this. My understanding is that we were going to replace the jumpstart stable-diffusion containers with our Rubikon container. In that case I think it might be confusing to also expose SD from here.